charts/spire/charts/spire-agent/templates/daemonset.yaml

```diff
@@ -115,17 +115,6 @@ spec:
               mountPath: /tpm
           imagePullPolicy: {{ .Values.nodeAttestor.tpmDirect.plugin.image.pullPolicy }}
         {{- end }}
-        - name: init
-          # This is a small image with wait-for-it, choose whatever image
-          # you prefer that waits for a service to be up. This image is built
-          # from https://github.com/vishnubob/wait-for-it
-          image: {{ template "spire-lib.image" (dict "image" .Values.waitForIt.image "global" .Values.global) }}
```

Comment:
The pr is looking pretty good.

I think its leaving behind a 'waitForIt' section in values.yaml. Could you remove that too please?

Reply:
Removed

Reply:
Oh. Found one more reference:
https://github.com/spiffe/helm-charts-hardened/blob/main/.github/tests/images.json#L10-L14

Reply:
Thanks, fixed